c_src/membrane_audio_mix_plugin/mixer.c

```diff
@@ -0,0 +1,210 @@
+#include "mixer.h"
+
+UNIFEX_TERM init(UnifexEnv *env, int channels, unsigned int format, int sample_rate)
+{
+    UNIFEX_TERM res;
+    State *state = unifex_alloc_state(env);
+    state->caps = unifex_alloc(sizeof(CapsAudioRaw));
+    state->caps->format = format;
+    state->caps->channels = channels;
+    state->caps->sample_rate = sample_rate;
+    state->sample_max = sample_max(state->caps);
+    state->sample_min = sample_min(state->caps);
+    state->sample_size = sample_size(state->caps);
+    state->queue = NULL;
+    state->queue_length = 0;
+    state->is_wave_positive = false;
+
+    res = init_result_ok(env, state);
+    return res;
+}
+
+void get_values(UnifexPayload **buffers, unsigned int buffers_length, int64_t *values, unsigned int *values_length, UnifexState *state)
```

Comment:
It would be nice to document these functions with docstrings like in https://github.com/membraneframework/membrane_common_c/blob/master/c_src/membrane_ringbuffer/ringbuffer.c